great_expectations_cloud/agent/cli.py

```diff
@@ -90,6 +90,10 @@ def main() -> None:
         )
     )
 
+    agent_logger = logging.getLogger(
```

Comment:
Some notes:
- This is set here and passed in bc logger must be aware of logging_cfg
- We want a logger name that is not the agent's __name__ to avoid submodules inheriting the INFO level